### ISSUE TITLE
Did the repo name change from ng-quill to ngQuill?

### ISSUE THREAD

**Issue by jziggas** [KillerCodeMonkey/ng-quill#68]
My `bower_components` directory has both now and my `index.html` and Gulpfile are all sort of confused.

**Comment by KillerCodeMonkey** [comment]
yeah, i removed it, because the git conventions changed in the time... so repos should be written in kebap-case (no uppercase letters)

SO ng-quill is the correct and new name :)

**Comment by jziggas** [comment]
Sorry I'm still a little confused. So I just `rm -rf`'d my bower_components folder and ran `bower install ng-quill#1.5.4`. This is the output:

```
> ~/project (dev) $ bower install ng-quill#1.5.4
bower ng-quill#1.5.4        not-cached https://github.com/KillerCodeMonkey/ngQuill.git#1.5.4
bower ng-quill#1.5.4           resolve https://github.com/KillerCodeMonkey/ngQuill.git#1.5.4
bower ng-quill#1.5.4          download https://github.com/KillerCodeMonkey/ngQuill/archive/1.5.4.tar.gz
bower ng-quill#1.5.4           extract archive.tar.gz
bower ng-quill#1.5.4      invalid-meta for:/var/folders/7y/dtxc4j3x3s76312w320yqrtmzsqbnh/T/ziggajt1/bower/10aff419f8a1f4fb06efb4ee1552cfb8-19263-i5bE0S/bower.json
bower ng-quill#1.5.4      invalid-meta The "name" is recommended to be lowercase, can contain digits, dots, dashes
bower ng-quill#1.5.4      invalid-meta The "main" field cannot contain minified files
bower ng-quill#1.5.4          mismatch Version declared in the json (1.5.3) is different than the resolved one (1.5.4)
bower ng-quill#1.5.4          resolved https://github.com/KillerCodeMonkey/ngQuill.git#1.5.4
bower quill#0.20.1              cached https://github.com/quilljs/quill.git#0.20.1
bower quill#0.20.1            validate 0.20.1 against https://github.com/quilljs/quill.git#0.20.1
```

And what ends up in my bower_components folder is `/ngQuill`

So is `ng-quill` really the correct name? If so why am I getting `ngQuill`?

**Comment by jziggas** [comment]
From a different project (we keep our bower components in a separate artifact that the main project downloads for version control purposes) it looks like I'm getting both dirs:

```
project (dev) $ bower install ng-quill#1.5.4 --save
bower ng-quill#1.5.4            cached https://github.com/KillerCodeMonkey/ngQuill.git#1.5.4
bower ng-quill#1.5.4          validate 1.5.4 against https://github.com/KillerCodeMonkey/ngQuill.git#1.5.4
bower keycloak        extra-resolution Unnecessary resolution: keycloak#1.8.1
bower ngInfiniteScroll extra-resolution Unnecessary resolution: ngInfiniteScroll#1.2.1
bower angular-sanitize extra-resolution Unnecessary resolution: angular-sanitize#^1.5.3
bower angular-cookie   extra-resolution Unnecessary resolution: angular-cookie#^4.1.0
bower Autolinker.js    extra-resolution Unnecessary resolution: Autolinker.js#~0.26.0
bower angular-ui-select extra-resolution Unnecessary resolution: angular-ui-select#^0.19.4
bower ngQuill#1.5.4              install ngQuill#1.5.4

ngQuill#1.5.4 bower_components/ngQuill
├── angular#1.5.9
└── quill#0.20.1
```

```
project (dev) $ git status
On branch dev
Your branch is behind 'origin/dev' by 1 commit, and can be fast-forwarded.
  (use "git pull" to update your local branch)
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   bower.json
	deleted:    bower_components/ng-quill/.bower.json
	deleted:    bower_components/ng-quill/.gitignore
	deleted:    bower_components/ng-quill/LICENSE.txt
	deleted:    bower_components/ng-quill/README.md
	deleted:    bower_components/ng-quill/bower.json
	deleted:    bower_components/ng-quill/karma.conf.js
	deleted:    bower_components/ng-quill/src/ng-quill.js
	deleted:    bower_components/ng-quill/src/ng-quill.min.js
	deleted:    bower_components/ng-quill/test/ng-quill-component.spec.js
	deleted:    bower_components/ng-quill/test/ng-quill-config.spec.js
	modified:   bower_components/ngQuill/.bower.json
	modified:   bower_components/ngQuill/.gitignore
	modified:   bower_components/ngQuill/LICENSE.txt
	modified:   bower_components/ngQuill/README.md
	modified:   bower_components/ngQuill/bower.json
	modified:   bower_components/ngQuill/src/ng-quill.js
	modified:   bower_components/ngQuill/src/ng-quill.min.js
```
